troyleach/divegalaxsea-ui
# Reef map

Changes that were made 
- Added reef map images and HTML

### How to test drive PR
- checkout branch 
- import `ReefMap.js` into `App.js` 
- replace component `<Home />` to `<ReefMap />`
- you should see 3 images across the top with the reef map in the middle of the page
